dlk/scripts/dlk_script_wrapper.sh

```diff
@@ -208,12 +214,15 @@ then
 	
 	make_each_target lm_x86 lm_x86.elf ${OUTPUT_DATA_DIR}
 	make_each_target lm_arm lm_arm.elf ${OUTPUT_DATA_DIR}
+	make_each_target lm_aarch64 lm_aarch64.elf ${OUTPUT_DATA_DIR}
 	make_each_target lm_fpga lm_fpga.elf ${OUTPUT_DATA_DIR}
 	make_each_target lib_x86 lib_x86.so ${OUTPUT_DATA_DIR}
 	make_each_target lib_arm lib_arm.so ${OUTPUT_DATA_DIR}
+	make_each_target lib_aarch64 lib_aarch64.so ${OUTPUT_DATA_DIR}
 	make_each_target lib_fpga lib_fpga.so ${OUTPUT_DATA_DIR}
 	make_each_target ar_x86 libdlk_x86.a ${OUTPUT_DATA_DIR}
 	make_each_target ar_arm libdlk_arm.a ${OUTPUT_DATA_DIR}
+	make_each_target ar_aarch64 libdlk_aarch64.a ${OUTPUT_DATA_DIR}
```

Comment:
This script file is not need anymore. We should delete this file, and `auto_script.md` also.`blueoul/blueoil_convert.py` is covered what this script do.

Reply:
Could you clarify the scope to cover with this PR?

Reply:
OK. We should delete these files in another PR 👍